### ISSUE TITLE
Brush updating

### ISSUE THREAD

**Issue by csoneson** [csoneson/iSEE#106]
With the Allen data set as an example:
```
1. Column data plot: Receive brush from Gene expression plot (Transparent effect)
2. Gene expression plot: Receive brush from Reduced dimension plot (Transparent effect)
3. Brush in Reduced dimension plot
4. Brush in Gene expression plot a region that contains both transparent and non-transparent points
5. Gene expression plot: Change Brush effect to Restrict
```
This does not update the brush transmission to the Column data plot until the brush in the Gene expression plot is moved around or otherwise updated

**Comment by LTLA** [comment]
I see this. That's funny - observers should have triggered - I'll have a look.

**Comment by LTLA** [comment]
Should be fixed in 1a88ed898477704a6e61855020261704e95cf308.

**Comment by csoneson** [comment]
Yep, works!
